### ISSUE TITLE
Semantically connect the tab list to the right side bar

### ISSUE THREAD

**Issue by MarcoZehe** [vector-im/riot-web#11319]
Right now, the tab list (the div with class mx_HeaderButtons), which contains the tabs pertaining to the right side bar, is part of the main header wrapper. But semantically, it belongs to the aside that contains the side bar. It should be moved into that side bar to create a DOM-wise and screen-reader logical connection to that side bar, since from the current layout, it is not clear that these tabs affect the side bar, not the main view of the room.
